apps/src/code-studio/components/libraries/LibraryPublisher.jsx

```diff
@@ -25,6 +25,11 @@ const styles = {
     width: 20,
     height: 20
   },
+  selectAllFunctionsLabel: {
+    margin: 0,
+    fontSize: 20,
+    fontFamily: '"Gotham 5r", sans-serif'
```

Comment:
Rather than setting the `fontFamily`, would it make more sense to set the `fontWeight`? I'm slightly concerned that explicitly setting the `fontFamily` could cause this to display as a different looking text font on certain browsers.

Reply:
i changed the fontFamily because that's the convention i've seen throughout apps/ -- are we using fontWeight as the convention instead now?

Reply:
Ahh I didn't realize we bundle Gotham as part of our app. This makes more sense now. It seems a little odd to me that we have `sans-serif` as a fallback here without making it bold. But I don't think this is a ship blocker for this.